src/clientAuth/bearerBody.js

```diff
@@ -8,7 +14,6 @@ class BearerBody extends ClientAuthnMethod {
   }
 
   /**
-   * More complicated logic then I would have liked it to be
    *@param {*} cis Request class instance
```

Comment:
You cannot use * param. be more specific (`Request`?). Same below.

Reply:
Fixed